src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -227,6 +227,8 @@ public class SscsCaseData implements CaseData {
     @DocumentLinkMustBePdf(message = "You need to upload PDF documents only")
     private DocumentLink adjournCasePreviewDocument;
     private String adjournCaseGeneratedDate;
+    @JsonProperty("isAdjournmentInProgress")
+    private YesNo isAdjournmentInProgress;
```

Comment:
You dont need @JsonProperty is the field name matches that on definintions

Reply:
Ideally this allong with all the other adjournCase fields should be in  @JsonUnwrapped class, e.g.
https://github.com/hmcts/sscs-common/pull/1353/files#diff-abf0148d833c53c177405baf9e103cf7b73cc3b572ccb2c72bbeb95f1c48055aL330-L332

Reply:
everything has been moved into Adjournment class